### ISSUE TITLE
Question about Large Clipfreq

### ISSUE THREAD

**Issue by ZetangForward** [XueZeyue/DanceGRPO#17]
Hi, thanks for your great work. I use your code to train the Flux model; however, I found a large clip frequency of advantage.

As shown in the code below:

https://github.com/XueZeyue/DanceGRPO/blob/5cdcf84c48ac85267bf458c7bf8deca35326f67e/fastvideo/train_grpo_flux.py#L496-L504

I train around 100 steps and print the `ratio` value, and found the `ratio` value always around 0.7~0.8. 

![Image](https://github.com/user-attachments/assets/0e84c2e7-a35d-4b29-aa84-7493fd26e882)

Thus, it is frequently clipped by the `torch.clamp` function, leading to a large clip frequency:

![Image](https://github.com/user-attachments/assets/f1ea6b62-6f00-4197-8312-64273c7c7a34)
 
The reward value even got lower during the training:

![Image](https://github.com/user-attachments/assets/3ea0f8f1-f3f8-4dde-a1bc-69b7f23532c6)

Have you ever met this problem?

**Comment by ZetangForward** [comment]
> Sorry, do you use this code to train the model? I have checked many times. If I use 8, 16, or 32 GPUs to train, the ratio is around 1.0...So could you provide your detailed settings?
> 
> The reason why I print the **ratio** for each iteration is: The ratio must be **1.** before the first update. This is a good way to debug.
> 
> I just launched a debug trial, and here are my results in my repo:
> 
> ![Image](https://github.com/user-attachments/assets/3e4960a3-3d33-497e-a245-64d132b924ee)
> 
> The ratio is **1.**, are you building your own models?

Yes, I am training my own models with your code, and I log the `ratio` for the first step. 
My mini-batch size is set as 6 (i.e., rollout 6 times, just for debugging), and for each `advantage score`, for the first denoising step, the `ratio` is:

![Image](https://github.com/user-attachments/assets/6aa93cf2-da06-4bc4-9920-9ead12c6f846)

**Comment by XueZeyue** [comment]
It should have the ratio 1, because we **only** introduce new noise in the rollout.
In the training process, we use the same noise and calculate the probability of this noise.
Before the update, the probability must be the same, so the ratio should be 1.

**Comment by ZetangForward** [comment]
> It should have the ratio 1, because we **only** introduce new noise in the rollout and obtain the prev_sample. In the training process, we don't sample new noise but just calculate the probability of this prev_sample. Before the update, the probability must be the same, so the ratio should be 1. For example, if the advantage is larger than zero, the model should increase the probability of prev_sample, so we should increase the ratio.
> 
> ![Image](https://github.com/user-attachments/assets/5d976036-9434-4d6d-9220-b4cec03f4b38) as shown in this code.

got it, I am checking my code now, thx for your reply !

**Comment by SXUSongJH** [comment]
> > It should have the ratio 1, because we **only** introduce new noise in the rollout and obtain the prev_sample. In the training process, we don't sample new noise but just calculate the probability of this prev_sample. Before the update, the probability must be the same, so the ratio should be 1. For example, if the advantage is larger than zero, the model should increase the probability of prev_sample, so we should increase the ratio.
> > ![Image](https://github.com/user-attachments/assets/5d976036-9434-4d6d-9220-b4cec03f4b38) as shown in this code.
> 
> got it, I am checking my code now, thx for your reply !

@ZetangForward 请问你解决这个问题了吗？
